pkgs/servers/nfd/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, ndn-cxx
+, libpcap
+, fetchFromGitHub
+, boost
+, pkgconfig
```

```suggestion
, pkg-config
```
pkgconfig is a compatibility alias (an artifact from before the nix language supported hyphens in names), pkg-config is preferred